build.gradle

```diff
@@ -88,6 +88,8 @@ dependencies {
         exclude group: 'com.google.cloud', module: 'google-cloud-firestore'
         exclude group: 'com.google.cloud', module: 'google-cloud-storage'
     }
+    implementation 'com.hivemq:hivemq-mqtt-client:1.3.0'
```

Comment:
And one last thing. Can you please move this above firebase and also remove the empty line.

Reply:
done